From 5e8f8aac9990b8f4ee13316be7345ed2d6e1fc2d Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Thu, 21 Jul 2011 16:39:26 +0800
Subject: [PATCH 08/10] ENGR00153459 Integrate Huawei MU509 update

only build ip-up/down without PREBUILT_3G_MODEM_RIL set

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 android/Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/android/Android.mk b/android/Android.mk
index 7de14a0..69f68a9 100755
--- a/android/Android.mk
+++ b/android/Android.mk
@@ -15,6 +15,7 @@
 #
 
 LOCAL_PATH := $(call my-dir)
+ifneq ($(PREBUILT_3G_MODEM_RIL),true)
 include $(CLEAR_VARS)
 
 LOCAL_SRC_FILES := ip-up-ppp0.c
@@ -32,3 +33,4 @@ LOCAL_MODULE := ip-down-ppp0
 LOCAL_MODULE_PATH := $(TARGET_OUT_ETC)/ppp
 LOCAL_MODULE_TAGS := optional
 include $(BUILD_EXECUTABLE)
+endif
-- 
1.8.0

